akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -55,7 +55,7 @@ class Member private[cluster] (
    * cluster. A member that joined after removal of another member may be
    * considered older than the removed member. Note that is only makes
    * sense to compare with other members inside of one data center (upNumber has
-   * a higher risk of being reused across data centers).
+   * a higher risk of being reused across data centers). // TODO should we enforce this to compare only within DCs?
```

Comment:
That's an interesting question. We could try. I'm thinking if it would break anything when rolling update from old Akka version to this, but that might work.
Let's do that in a separate PR, if we do it.